plex-jail.sh

```diff
@@ -98,8 +121,18 @@ fi
 iocage exec "${JAIL_NAME}" crontab /configs/update_packages
 iocage fstab -r "${JAIL_NAME}" "${CONFIGS_PATH}" /configs nullfs rw 0 0
 iocage exec "${JAIL_NAME}" rm -rf /configs
-iocage restart "${JAIL_NAME}"
+if [ $HW_TRANSCODE_RULESET -ne 0 ]; then
+   iocage exec "${JAIL_NAME}" pw groupmod -n video -m plex
+fi
+if [ -z "${PLEX_MEDIA_PATH}" ]; then
+  iocage stop "${JAIL_NAME}"
+else
+  iocage restart "${JAIL_NAME}"
+fi
```

Comment:
Why stop the jail if PLEX_MEDIA_PATH isn't set?  Is it on the assumption that the user would be adding the mount point through the GUI (which inexplicably requires the jail be stopped)?